### PR TITLE
Next

### PR DESCRIPTION
**Pull request by benghaziboy**
@mparent61 @MichaelButkovic 

binning next result sets. 

can configure for larger & smaller binning values.

### CODE REVIEW

File: djes/search.py [hunk 1]
```diff
@@ -66,12 +67,15 @@ def __next__(self):
...
         # Tracks index on the search object so we can iterate across multiple queries.
-        start, end = self.position, self.position + 1
+        size = self._extra.get("size", 10)
+        from_ = self.position
         self.position += 1
-        try:
-            return self[start:end][0]
-        except IndexError:
-            raise StopIteration
+        if from_ + 1 > len(self._results):
```
Comment (mparent61) [inline djes/search.py]: simpler: `if from_ >= len(self._results)`

File: djes/search.py [hunk 2]
```diff
@@ -66,12 +67,15 @@ def __next__(self):
 
     def next(self):
         # Tracks index on the search object so we can iterate across multiple queries.
-        start, end = self.position, self.position + 1
+        size = self._extra.get("size", 10)
```
Comment (mparent61) [inline djes/search.py]: Is 10  the page size? Maybe make this a class variable/setting?

File: djes/search.py (hunk 1 above)
Comment (benghaziboy) [inline djes/search.py]: did have this,
But if `from_=0` you will end up with an indexing error

File: djes/search.py (hunk 2 above)
Comment (benghaziboy) [inline djes/search.py]: it is a class variable setting. 

but it assigns it to _extra

**Comment by mparent61** [comment]
Make "10" a DEFAULT_PAGE_SIZE class variable then :+1: 

I've been waiting for next() for a long time! (Now i need to run out and use it somewhere)

**Pull request merged**